sdk/framework/path.go

```diff
@@ -224,6 +224,22 @@ type DisplayAttributes struct {
 	// Action is the verb to use for the operation.
 	Action string `json:"action,omitempty"`
 
+	// OperationPrefix is a hyphenated lower-case string used to construct
+	// OpenAPI OperationID. It is typically the name of the plugin.
+	OperationPrefix string `json:"operationPrefix,omitempty"`
+
+	// OperationPrefix is a hyphenated lower-case string used to construct
+	// OpenAPI OperationID. It is typically an action to be performed
+	// (e.g. "generate", "sign", "login", etc.)
+	OperationVerb string `json:"operationVerb,omitempty"`
```

Comment:
what's the default if empty? we go off of http verb? (eg: POST requests are Write, etc)

Reply:
The default will be the Operation enum's string representation (e.g. read, delete, list, etc.). The only exception to that is "update" is replaced with "write" since it's more appropriate in most cases. The logic is at the end of `constructOperationID`